src/components/map/Map.js

```diff
@@ -18,30 +19,29 @@ const containerStyle = {
 };
 
 const center = {
-  lat: 40.745255,
-  lng: -74.034775,
+  lat: 40.712255,
+  lng: -74.044775,
 };
 
 const getPixelPositionOffset = (width, height) => ({
   x: -(width / 2),
   y: -(height / 2),
 });
 
-const Map = ({ deals }) => {
+const Map = ({ day, filters, filterResult }) => {
   const { isLoaded } = useJsApiLoader({
     id: "google-map-script",
     googleMapsApiKey: process.env.REACT_APP_GOOGLE_MAPS_API_KEY,
   });
 
-  const [selectedPlace, setSelectedPlace] = React.useState(0);
+  const [selectedPlace, setSelectedPlace] = React.useState(-1);
   const [map, setMap] = React.useState(null);
   const [markerLocations, setMarkerLocations] = React.useState([]);
 
   const onLoad = React.useCallback(async function callback(map) {
-    const url = `${process.env.REACT_APP_API_URL}/api/map/locations${deals
-      .map((data, index) => `${index === 0 ? "?" : "&"}deals=${data._id}`)
-      .toString()
-      .replaceAll(",", "")}`;
+    const url = `${process.env.REACT_APP_API_URL}/api/map/locations?day=${day}&${filterResult}${filters.hobo ? "&city=Hoboken" : ""}${
```

Comment:
nit: doesn't need to be addressed now but there needs to be a better way to create this url for active, and city